pyproject.toml

```diff
@@ -25,7 +25,7 @@ profile = "black"
 
 [tool.flake8]
 max-line-length = 88
-ignore = ["E302", "E305", "E306", "E402", "E731", "F401", "F403", "F841", "W503"]
+ignore = ["E203", "E302", "E305", "E306", "E402", "E731", "F401", "F403", "F841", "W503"]
```

Comment:
Black wants space before colon?

Reply:
Yup -> https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8

Reply:
Might be worth using `extend-ignore` instead of `ignore` at some point in the future.